GraphNeuralNetworks/src/layers/temporalconv.jl

```diff
@@ -758,7 +758,7 @@ EvolveGCNO(args...; kws...) = GNNRecurrence(EvolveGCNOCell(args...; kws...))
 
 
 """
-    TGCNCell(in => out; kws...)
+    TGCNCell(in => out; act = relu, kws...)
```

Comment:
```suggestion
    TGCNCell(in => out, act = relu; kws...)
```
Let's have the activation as a second positional argument as customary, instead of keyword argument